spec/index/index_spec.rb

```diff
@@ -102,6 +120,11 @@
       subject { Daru::Index.new ('a'..'z').to_a  }
       its(:inspect) { is_expected.to eq "#<Daru::Index(26): {a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t ... z}>" }
     end
+
+    context 'index with name' do
+      subject { Daru::Index.new ['one', 'two', 'three'], name: 'number'  }
+      its(:inspect) { is_expected.to eq "#<Daru::Index: number(3): {one, two, three}>" }
```

Comment:
Thanks @zverok for review. I have changed it in https://github.com/SciRuby/daru/pull/323/commits/56b84d8c73e7afd92e05292f6ffee2b785f4c28b